index.html

```diff
@@ -202,8 +202,8 @@ <h3>What happens to unresolved issues marked *-needs-resolution?</h3>
 
 
 <section id="horizontal_review_boards">
-<h2><a href="#horizontal_review_boards">Horizontal review boards</a></h2>
-<p>The horizontal groups maintain repositories containing issues that track those raised in the WG repos. Key information about the current state of those tracker repositories is reflected in a <a rel="nofollow" class="external text" href="https://w3c.github.io/horizontal-issue-tracker/index">set of tracker boards</a>, one per horizontal function. The board points to WG issues and the corresponding tracker issues, and groups issues by specification.
+<h2><a href="#horizontal_review_boards">Issue trackers</a></h2>
+<p>The horizontal groups maintain repositories containing issues that track those raised in the WG repos. You can see a list of tracked issues on the <a rel="nofollow" class="external text" href="https://w3c.github.io/horizontal-issue-tracker/index">tracker boards</a>, one per horizontal area.
```

Comment:
Actually, and importantly, there are links to both the tracked issues (ie. the ones in the non-HR-group's repo) and tracker issues (the ones in the HR-group repo). So, if you're trying to remove the details of the description, you might need to say something like "You can find a list of links on the..." 

And then you have 'tracker boards', which i thought you wanted to replace with 'issue trackers'.

If you make that change here also, then perhaps you should just say:
"See the issue trackers for links."